AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/EditorEntityBoolProperty.py

```diff
@@ -0,0 +1,15 @@
+from editor_python_test_tools.editor_component.EditorEntityComponentProperty import EditorEntityComponentProperty
+from editor_python_test_tools.editor_entity_utils import EditorComponent
+
+class EditorEntityIntegerProperty(EditorEntityComponentProperty):
+    """
+    Defines the behaviors for interacting with an Editor Entity boolean property
```

Comment:
nit: It looks like maybe the comment needs to be updated here if this is `IntegerProperty` and not `BoolProperty`? (or maybe the type name is incorrect? Should it be `EditorEntityBoolProperty`?)

Reply:
yep this is a typo